### PR TITLE
Updated Tasks topic for SEO Report Items

### PR DESCRIPTION
**Pull request by JRAlexander**
Updated Tasks topic for SEO Report Items as part of HackAdoc

### CODE REVIEW

File: docs/machine-learning/resources/tasks.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
 ---
-title: Machine learning tasks
-description: Explore the different machine learning tasks supported in ML.NET.
+title: Machine learning tasks - ML.NET | Microsoft Docs
```
Comment (mairaw) [inline docs/machine-learning/resources/tasks.md]: don't add | Microsoft Docs. this is automatically added by the build

File: docs/machine-learning/resources/tasks.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
+description: Explore the different machine learning tasks and associated learners that are supported in ML.NET.
+ms.custom: seodec18
+ms.manager: cgronlun
```
Comment (mairaw) [inline docs/machine-learning/resources/tasks.md]: this metadata field is not valid

https://review.docs.microsoft.com/en-us/help/contribute/metadata-attributes?branch=master

**Review by mairaw — APPROVED** [review]
LGTM but left some comments for you to consider

File: docs/machine-learning/resources/tasks.md (hunk 1 above)
Comment (JRAlexander) [reply docs/machine-learning/resources/tasks.md]: oh, it got called out. Will remove!

**Pull request merged**